components/sysutils/phoronix-test-suite/phoronix-test-suite.p5m

```diff
@@ -25,9 +25,9 @@ license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
 file path=etc/bash_completion.d/phoronix-test-suite
 file path=usr/bin/phoronix-test-suite
-#file path=usr/lib/systemd/system/phoromatic-client.service
-#file path=usr/lib/systemd/system/phoromatic-server.service
-#file path=usr/lib/systemd/system/phoronix-result-server.service
+file path=usr/lib/systemd/system/phoromatic-client.service
+file path=usr/lib/systemd/system/phoromatic-server.service
+file path=usr/lib/systemd/system/phoronix-result-server.service
```

Comment:
Why you uncommented these files?  You said at the other place that you [was trying to avoid installing (unnecessary) systemd files](https://github.com/OpenIndiana/oi-userland/pull/9159#discussion_r1559613765) so why you add such _(unnecessary) systemd files_ to the manifest?

Reply:
Because I was told not to manually edit the `sample-manifest.p5m`, and @AndWac has had difficulty replicating my builds. So I figured I should try not commenting out *anything* until we can get everything working. Those systemd services are just text files, so it really doesn't add a lot of overhead

Reply:
Yes, you should never manually edit `sample-manifest.p5m`.  Regular manifests are free to edit as needed.

By definition, the removal (including commenting out) any `file` actions from manifests is making the build replication easier or same while the addition of `file` actions is making the build replication harder or same. 

Either way it makes no sense to uncomment those files.